search.java

```diff
@@ -0,0 +1,39 @@
+package searchFile;
+/*
+ * Program that searches a specified text file where a specified word occurs and prints out the whole line.
+ * 
+ * @author Kadriye Yildiz 
+ */
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Scanner;
+
+public class search {
+	
+	public static void main(String[] args) throws IOException{
+		
+		Scanner sc = new Scanner(System.in); 
```

Comment:
Looks good besides maybe the formatting on a few rows?

Reply:
okay, thanks!